clients/cloud/c/common.c

```diff
@@ -129,7 +129,7 @@ int create_topic (rd_kafka_t *rk, const char *topic,
 
         fprintf(stderr, "Creating topic %s\n", topic);
 
-        newt = rd_kafka_NewTopic_new(topic, num_partitions, replication_factor,
+        newt = rd_kafka_NewTopic_new(topic, num_partitions, -1,
```

Comment:
`.., num_partitions, -1/*use broker default RF*/,`

Reply:
Another idea could be to use a const

```
/* Signals to use the broker default */
const int DEFAULT_REPLICATION_FACTOR = -1;
```

and then
```
        newt = rd_kafka_NewTopic_new(topic, num_partitions, DEFAULT_REPLICATION_FACTOR,
                                     errstr, sizeof(errstr));
```